### PR TITLE
[Site] Change color and style of menu items --signoff

### LINKED ISSUE #124: [Site] Change color and style of menu items
**Current State of the GetNighthawk.dev site**
The current implementation of the following sections should be corrected to follow the design specification:
1. the FAQs section of [getnighthawk.dev](https://getnighthawk.dev/)

FAQs

2. The sidebar of [Getnighthawk Docs](https://getnighthawk.dev/docs)

Docs SideNav

**Desired State of the GetNighthawk.dev site**

The design should be similar to this for the FAQs section: 
FAQs

Similar design should also be implemented for the Docs side-navbar.

For convenience, you can check out the [Figma design](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs).

---

**Contributor Resources**

- See the [Contributing to getnighthawk.dev Website](https://github.com/layer5io/getnighthawk/blob/master/CONTRIBUTING.md)
- See the Project Site section of [GetNighthawk project doc](https://docs.google.com/document/d/1lHfMo4iIx2WXFZIspfHyxTsPR1T63_2IV5NUkgxoo0w)

### PR DESCRIPTION
**Pull request by NikhilSharma03**
Signed-off-by: Nikhil <nikhilsharmamusic2000@gmail.com>

**Description**
Updated faq color and style as in the Figma design

## After Changes
FAQ

This PR fixes #124 

**Notes for Reviewers**
@JubayerJoy Can you please confirm the changes so I can work on the " The sidebar of Getnighthawk Docs " section too? Thank you.

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by vinayaksh42** [comment]
Nice changes @NikhilSharma03

**Review by JubayerJoy — COMMENTED** [review]
Nice work @NikhilSharma03, the implementation is near perfect as the design 🎈 

Needs some fixes,
1. The `font-family` of this section is specified to be `open sans` in the design.
2. There are some issues with the smaller screen:
Screenshot (216)
 #### Expected: 
 
Screenshot (217)

Good luck 👍

**Comment by NikhilSharma03** [comment]
Thank you for the review @JubayerJoy. I will work on that.

**Comment by JubayerJoy** [comment]
@NikhilSharma03 You can do a similar implementation for the Docs menu, it can be done in this or separate PR at your convenience. Thank you!

**Comment by NikhilSharma03** [comment]
Yes for sure @JubayerJoy. I was just making sure if these changes are correct so I can implement these changes into the docs too.
I guess a different PR will be more right since the changes belong to a different section.

**Comment by NikhilSharma03** [comment]
## Changes Made
- Updated font-family to Open Sans for this section
- Fixed padding of the menu

## After Changes
Screenshot from 2021-06-02 20-17-53

@JubayerJoy Can you please review the changes?

File: docs/_sass/faq.scss [hunk 1]
```diff
@@ -27,4 +66,12 @@
     button{
         margin: 20px 0;
     }
+}
+
+@media(max-width:1100px){
+    .container-faq .col-lg-8{
+        padding-left: 4rem;
+        padding-right: 4rem;
```
Comment (JubayerJoy) [inline docs/_sass/faq.scss:74]: Let's trim down the padding a bit more for matching this section with the upper ones.

```suggestion
        padding-left: 2rem;
        padding-right: 2rem;
```

**Review by JubayerJoy — COMMENTED** [review]
@NikhilSharma03 Really nice work 🚀
Just a small change....

File: docs/_sass/faq.scss (hunk 1 above)
Comment (NikhilSharma03) [reply docs/_sass/faq.scss:74]: Yeah Sure.

**Comment by NikhilSharma03** [comment]
@JubayerJoy Can you please review? Thank you.

**Review by JubayerJoy — APPROVED** [review]
LGTM 🎈

**Comment by NikhilSharma03** [comment]
Thank you @JubayerJoy. @vinayaksh42 Can you please review the changes? Thank you.

**Review by vinayaksh42 — APPROVED** [review]

**Comment by leecalcote** [comment]
Some good work happening here.

**Comment by NikhilSharma03** [comment]
@leecalcote 😂🚀🚀

**Pull request merged**